### PR TITLE
Xen to oVirt

### PR DESCRIPTION
**Pull request by mureinik**
Changes proposed in this pull request:
- Move the Xen To oVirt page under virt
- Fix markdown syntax
- Fix typos

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @mureinik

This pull request needs review by: @shaharha / @ahadas

### CODE REVIEW

**Comment by bproffitt** [comment]
@ahadas @shaharha Could you review this so we can merge? Thanks!

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 1]
```diff
@@ -8,29 +8,30 @@ wiki_revision_count: 1
...
 # Importing Xen on EL 5.x VMs to oVirt
------------------------------------------------------------
+
 ## Summary
 oVirt has the ability to import VMs from other hypervisor including **Xen** on EL 5.x (not yet for Citrix Xen)
-The Import process uses [virt-v2v][1] (under the "INPUT FROM EL 5 XEN" section) which explain the prerequisites that are needed in order to import Xen VMs.
-[1]: http://libguestfs.org/virt-v2v.1.html
-This feature is available from version 4.0
+The Import process uses
+[virt-v2v](http://libguestfs.org/virt-v2v.1.html) (under the "INPUT FROM EL 5 XEN" section) which explain the prerequisites that are needed in order to import Xen VMs.
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: why breaking line 18?

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 2]
```diff
@@ -8,29 +8,30 @@ wiki_revision_count: 1
...
 In order to import VMs password-less SSH access has to be enabled between VDSM host and the Xen host.
 The following steps needed to be done at the VDSM host:
-- Generate ssh kes for vdsm user ```sudo -u vdsm ssh-keygen```
-- Copy vdsms public key to the Xen host```sudo -u vdsm ssh-copy-id user@xenhost```
-- Login to the remote host (in order to test the connection and add the Xen host to the known_hosts file ```sudo -u vdsm ssh user@xenhost```
-- Exit the remote Xen host ```logout```
 
-We are logging to the xenhost after we copy the ssh key in order to check that the ssh-copy-id succeeded and in order to set the host public key in the ~/.ssh/known_hosts.
+- Generate ssh keys for vdsm user `sudo -u vdsm ssh-keygen`
+- Copy vdsm's public key to the Xen host`sudo -u vdsm ssh-copy-id user@xenhost`
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: missing space?

File: source/develop/release-management/features/virt/XenToOvirt.html.md [hunk 3]
```diff
@@ -8,29 +8,30 @@ wiki_revision_count: 1
...
+- Login to the remote host (in order to test the connection and add the Xen host to the `known_hosts` file `sudo -u vdsm ssh user@xenhost`
+- Exit the remote Xen host `logout`
+
+We are logging to the xenhost after we copy the ssh key in order to check that the ssh-copy-id succeeded and in order to set the host public key in the `~/.ssh/known_hosts`.
```
Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: Xen host?

Comment (ahadas) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: host's public key

File: source/develop/release-management/features/virt/XenToOvirt.html.md (hunk 1 above)
Comment (mureinik) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: Done.

File: source/develop/release-management/features/virt/XenToOvirt.html.md (hunk 2 above)
Comment (mureinik) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: yup, will fix

File: source/develop/release-management/features/virt/XenToOvirt.html.md (hunk 3 above)
Comment (mureinik) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: will fix

Comment (mureinik) [inline source/develop/release-management/features/virt/XenToOvirt.html.md]: will fix

**Comment by mureinik** [comment]
@ahadas Fixed according to your comments (thanks, BTW!). Can you take another look please?

**Pull request merged**